source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -27,6 +27,19 @@ Common::Redis::RespValuePtr Utility::makeError(const std::string& error) {
 
 namespace {
 
+// nullPoolCallbacks is used for requests that must be filtered and not redirected such as "asking".
+DoNothingPoolCallbacks nullPoolCallbacks;
+
+// Create an asking command request.
+Common::Redis::RespValue askingRequest() {
```

Comment:
Instead of constructing a new `RespValue` each time, this can be a static value that you construct once and then use every time I think?

Reply:
good idea